src/contracts/business-center/business-center.spec.ts

```diff
@@ -54,69 +45,75 @@ describe('Business Center', function test() {
-    const adminFactory = await loader.loadContract('BusinessCenterFactory', adminFactoryContractAddress);
-    const address = await executor.executeContractTransaction(
+    const adminFactoryContractAddress = await runtimes[0].nameResolver.getAddress(
+      adminFactoryEnsDomain,
+    );
+    const adminFactory = await runtimes[0].contractLoader.loadContract(
+      'BusinessCenterFactory',
+      adminFactoryContractAddress,
+    );
+    const address = await runtimes[0].executor.executeContractTransaction(
       adminFactory,
       'createContract',
       {
-        from: accounts[0],
+        from: identity0,
         gas: 5000000,
         event: { target: 'BusinessCenterFactory', eventName: 'ContractCreated' },
         getEventResult: (event, args) => args.newAddress,
       },
-      nameResolver.namehash(ensDomain),
+      runtimes[0].nameResolver.namehash(ensDomain),
       config.nameResolver.ensAddress,
     );
-    const bc = loader.loadContract('BusinessCenterInterface', address);
+    const bc = runtimes[0].contractLoader.loadContract('BusinessCenterInterface', address);
     const storageAddress = '0x0000000000000000000000000000000000000000';
-    await executor.executeContractTransaction(
+    await runtimes[0].executor.executeContractTransaction(
       bc,
       'init',
-      { from: accounts[0], autoGas: 1.1 },
+      { from: identity0, autoGas: 1.1 },
       storageAddress,
       joinSchema,
     );
-    await executor.executeContractTransaction(bc, 'join', { from: accounts[0], autoGas: 1.1 });
+    await runtimes[0].executor.executeContractTransaction(bc, 'join', { from: identity0, autoGas: 1.1 });
     return bc;
   }
 
   before(async () => {
-    executor = await TestUtils.getExecutor(web3);
-    executor.eventHub = await TestUtils.getEventHub(web3);
-    loader = await TestUtils.getContractLoader(web3);
-    nameResolver = await TestUtils.getNameResolver(web3);
-    ensDomain = nameResolver.getDomainName(config.nameResolver.domains.businessCenter);
+    runtimes = await Promise.all(
+      accounts.slice(0, 2).map((account) => TestUtils.getRuntime(account, null, { useIdentity })),
+    );
+    identity0 = runtimes[0].activeIdentity;
+    identity1 = runtimes[1].activeIdentity;
+    ensDomain = runtimes[0].nameResolver.getDomainName(config.nameResolver.domains.businessCenter);
     businessCenter = await createBusinessCenter(0);
-    const isMember = await executor.executeContractCall(
-      businessCenter, 'isMember', accounts[2], { from: accounts[2], gas: 3000000 },
+    const isMember = await runtimes[0].executor.executeContractCall(
+      businessCenter, 'isMember', identity0,
     );
     if (!isMember) {
-      await executor.executeContractTransaction(
-        businessCenter, 'join', { from: accounts[2], autoGas: 1.1 },
+      await runtimes[0].executor.executeContractTransaction(
+        businessCenter, 'join', { from: identity0, autoGas: 1.1 },
```

Comment:
shouldn't this be accounts[2] ? because identity0 is creating the businesscenter
```suggestion
        businessCenter, 'join', { from: identity1, autoGas: 1.1 },
```